tutorials/lime_text.ipynb

```diff
@@ -180,8 +180,8 @@
    ],
```

Comment:
<p>Much better already.</p><p>"while words against the selected class - in blue." I think would read even easier if it were:</p><p>"while words against the selected class are highlighted in blue."</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dianna-ai/dianna/pull/210/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>